docs/criteria/accelerator-stages.md

```diff
@@ -9,51 +9,41 @@ Current supported data accelerators (DuckDB, Postgres, SQlite, Arrow) release st
 ### Feature set complete
 
 - [ ] Streaming when accelerating/loading
-- [ ] Index
+- [ ] Streaming when querying
+- [ ] Supports primary keys and indexes
+- [ ] Supports federation in a single dataset
+- [ ] Supports push-down federation across multiple datasets within the same accelerator
 - [ ] On conflict behaviors - Drop/Upsert
-- [ ] File mode for SQLite and DuckDB
+- [ ] File mode for supported accelerator engines, i.e. SQLite, DuckDB, etc.
 - [ ] Common/core arrow data type supports: Null, all Int types, all Float types, Time32, Time64, Timestamp (with/without TZ), Date32, Date64, Duration, Interval, Binary/LargeBinary/FixedSizeBinary, Utf8/LargeUtf8, List/FixedSizeList/LargeList, Struct, Decimal128/Decimal256
 - [ ] No known major bugs/issues
 
 ### Test Coverage
 
 - [ ] End-to-end test to cover accelerating TPCH-SF1 data from S3 and benchmarking TPCH queries (official and simple).
-  - file mode if applicable
-  - memory mode if applicable
+  - [ ] File mode
+  - [ ] Embedded memory mode
 - [ ] Integration tests to cover accelerating data from S3 parquet, MySQL, Postgres with arrow types: Null, all Int types, all Float types, Time32, Time64, Timestamp (with/without TZ), Date32, Date64, Duration, Interval, Binary/LargeBinary/FixedSizeBinary, Utf8/LargeUtf8, List/FixedSizeList/LargeList, Struct, Decimal128/Decimal256
```

Comment:
Should split the data connectors to sub tasks. Is this the source of truth for data types?

Reply:
> Should split the data connectors to sub tasks

👍

> Is this the source of truth for data types?

Its the list of types we decided to support, so its the source of truth for that. There are other Arrow types/esoteric DB types that we don't claim to support (i.e. Postgres geometry types, Arrow StringView, etc)